core/src/main/java/io/neow3j/protocol/core/HexUtils.java

```diff
@@ -0,0 +1,31 @@
+package io.neow3j.protocol.core;
+
+import io.neow3j.crypto.KeyUtils;
+import io.neow3j.utils.ArrayUtils;
+import io.neow3j.utils.Numeric;
+
+import java.math.BigInteger;
+
+public class HexUtils {
+
+    public static String reverse(String input) {
+        byte[] inputBytes = Numeric.hexStringToByteArray(input);
+        byte[] reversedBytes = ArrayUtils.reverseArray(inputBytes);
+        return Numeric.toHexStringNoPrefix(reversedBytes);
+    }
+
+    public static String hexToString(String input) {
```

Comment:
This method should be moved to the `Numeric` class, since it's a convenience method for the `Numeric.hexStringToByteArray()`.
Please, also, move the respective test to the `NumericTest` class.